configuration/editors.md

```diff
@@ -0,0 +1,188 @@
+
+- **Windows**: Extract the archive to a path of your choosing, e.g. `C:\designer`
+- **Linux**: Extract the archive to a path of your choosing, e.g. `/opt/designer`
+- **Max OSX**: Extract the archive to your applications folder
+
+### First Launch
+
+After launching the Designer executable you will see an empty configuration file pane on the top left.
+
+  ![SmartHome Designer standard view](images/smarthome-designer-10.png)
+
+Click on the "Select a configuration folder" icon at the top right of the configuration window and navigate to your openHAB configuration folder (containing `items`, `rules`, ...).
+
+  ![SmartHome Designer configuration folder selection](images/smarthome-designer-15.png)
+
+The configuration pane should now list the different configuration folders:
+
+  ![SmartHome Designer standard view with loaded configuration](images/smarthome-designer-20.png)
+
+### Usage
+
+You can now start to create and modify your configuration files.
+Changes are automatically loaded by the openHAB runtime.
+More details about the syntax of the different configuration files can be found in the [Configuration]({{base}}/configuration/index.html) chapter.
+
+Hint: Press `<Ctrl + N>` to create a new file.
+
+## Others
+
+### mcedit
+
+mcedit is an editor which comes with mc (Midnight Commander).
+
+#### Installing the syntax-files
+
+- Navigate to [openhab-mcedit](https://github.com/CWempe/openhab-mcedit) and follow the instructions in the repos readme file.
+
+### Notepad++
+
+Notepad++ Version 6.2 or above is required to support UDL2 (User Defined Language v2).
+
+#### Comments
+
+```shell
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"
+```
+http://notepad-plus-plus.org/news/notepad-6.2-release-udl2.html
```

Comment:
Also, any clue what's the meaning of these lines? The headline just says "comments"...